roachtest: import/experimental-direct-ingestion failed

SHA: https://github.com/cockroachdb/cockroach/commits/a850466dfc4f8eed9e7f758a61f60b120798410f

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=import/experimental-direct-ingestion PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1426564&tab=buildLog

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20190807-1426564/import/experimental-direct-ingestion/run_1
	cluster.go:1735,import.go:69,cluster.go:2078,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1565157871-25-n3cpu16:1 -- ./cockroach workload fixtures import tpcc --warehouses=1000 --experimental-direct-ingestion --checks=false returned:
		stderr:
		
		stdout:
		6  imported item (3s, 100000 rows, 0 index entries, 7.8 MiB)
		I190807 12:04:20.977855 89 ccl/workloadccl/fixture.go:516  imported new_order (33s, 9000000 rows, 0 index entries, 126 MiB)
		I190807 12:06:12.931598 87 ccl/workloadccl/fixture.go:516  imported history (2m24s, 0 rows, 0 index entries, 0 B)
		I190807 12:06:12.931596 91 ccl/workloadccl/fixture.go:516  imported stock (2m24s, 0 rows, 0 index entries, 0 B)
		I190807 12:06:12.931599 92 ccl/workloadccl/fixture.go:516  imported order_line (2m24s, 0 rows, 0 index entries, 0 B)
		I190807 12:06:12.931601 86 ccl/workloadccl/fixture.go:516  imported customer (2m24s, 0 rows, 0 index entries, 0 B)
		I190807 12:06:12.931603 88 ccl/workloadccl/fixture.go:516  imported order (2m24s, 0 rows, 0 index entries, 0 B)
		W190807 12:06:12.935425 1 ccl/workloadccl/fixture.go:569  error enabling automatic stats: dial tcp 127.0.0.1:26257: connect: connection refused
		Error: importing fixture: importing table stock: dial tcp 127.0.0.1:26257: connect: connection refused
		Error:  exit status 1
		: exit status 1
	cluster.go:2099,import.go:73,import.go:104,test_runner.go:691: unexpected node event: 1: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/51a6fdedf0ce1d1329d40d801a7deaf8206b6b07

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=import/experimental-direct-ingestion PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1428934&tab=buildLog

```
The test failed on branch=provisional_201908060405_v19.1.4, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20190807-1428934/import/experimental-direct-ingestion/run_1
	cluster.go:1735,import.go:69,cluster.go:2078,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1565218672-25-n3cpu16:1 -- ./cockroach workload fixtures import tpcc --warehouses=1000 --experimental-direct-ingestion --checks=false returned:
		stderr:
		
		stdout:
		workloadccl/fixture.go:516  imported warehouse (2s, 1000 rows, 0 index entries, 53 KiB)
		I190808 04:37:59.398115 74 ccl/workloadccl/fixture.go:516  imported new_order (37s, 9000000 rows, 0 index entries, 126 MiB)
		I190808 04:44:29.092791 77 ccl/workloadccl/fixture.go:516  imported order_line (7m7s, 0 rows, 0 index entries, 0 B)
		I190808 04:44:29.092791 73 ccl/workloadccl/fixture.go:516  imported order (7m7s, 0 rows, 0 index entries, 0 B)
		I190808 04:44:29.092792 72 ccl/workloadccl/fixture.go:516  imported history (7m7s, 0 rows, 0 index entries, 0 B)
		I190808 04:44:29.092810 71 ccl/workloadccl/fixture.go:516  imported customer (7m7s, 0 rows, 0 index entries, 0 B)
		I190808 04:44:29.092817 76 ccl/workloadccl/fixture.go:516  imported stock (7m7s, 0 rows, 0 index entries, 0 B)
		W190808 04:44:29.115786 1 ccl/workloadccl/fixture.go:569  error enabling automatic stats: dial tcp 127.0.0.1:26257: connect: connection refused
		Error: importing fixture: importing table stock: dial tcp 127.0.0.1:26257: connect: connection refused
		: signal: killed
	cluster.go:2099,import.go:73,import.go:104,test_runner.go:691: EOF

```

--- a/pkg/cmd/roachtest/import.go
+++ b/pkg/cmd/roachtest/import.go
@@ -97,7 +97,7 @@ func registerImportTPCC(r *testRegistry) {
 	})
 	r.Add(testSpec{
 		Name:       `import/experimental-direct-ingestion`,
-		MinVersion: `v19.1.0`,
+		MinVersion: `v19.2.0`,
 		Cluster:    makeClusterSpec(3, cpu(16)),
 		Timeout:    2 * time.Hour,
 		Run: func(ctx context.Context, t *test, c *cluster) {

